packages/evm/README.md

```diff
@@ -62,7 +59,6 @@ evm
   })
   .then((results) => {
     console.log(`Returned: ${results.returnValue.toString('hex')}`)
-    console.log(`gasUsed : ${results.gasUsed.toString()}`)
```

Comment:
Instead of deleting this, let's just print the correct value.

`console.log(`gasUsed: ${result.executionGasUsed.toString()}`)`